independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeployment.java

```diff
@@ -520,6 +522,10 @@ public BeanResolver getBeanResolver() {
         return beanResolver;
     }
 
+    public DelegateInjectionPointResolverImpl getDelegateInjectionPointResolver() {
```

Comment:
I don't think that we should expose an impl class, i.e. this method should return `BeanResolver` and the `DelegateInjectionPointResolverImpl` class should be package-private.